avocado/core/utils.py

```diff
@@ -69,7 +69,9 @@ def resolutions_to_tasks(resolutions, config):
     filter_by_tags = config.get("filter.by_tags.tags")
     include_empty = config.get("filter.by_tags.include_empty")
     include_empty_key = config.get('filter.by_tags.include_empty_key')
-    status_server = config.get('nrun.status_server.listen')
+    status_server = config.get('nrunner.status_server_uri')
+    if status_server is None:
+        status_server = config.get('nrun.status_server.listen')
```

Comment:
I am a bit confused by the introduction of `nrunner` namespace and its relation w.r.t. the `nrun` namespace. I guess both are temporary?

Reply:
So the "plugin that runs a test suite in the context of an Avocado job" is named `nrunner`.  You can see that with `avocado plugins`.  So, given that we have two runners (`runner` and `nrunner`), and that those options are `nrunner` specific, I proposed to use the `nrunner` prefix.  And `nrun` is supposed to be removed soon after this gets merged.

Reply:
I see, so similar to `runner` and `run` namespaces. Just to sum up: the final namespaces we expect to have are `run`, `runner, and `nrunner`, correct? It makes sense now.

Reply:
Yes!